src/video/video_driver.hpp

```diff
@@ -178,6 +178,11 @@ class VideoDriver : public Driver {
 		return ZOOM_LVL_OUT_4X;
 	}
 
+	virtual const char * GetInfoString() const
```

Comment:
Nitpicking: space after * seems random :)

PS: nice idea, this PR :D

Reply:
I honestly don't remember what the convention is, I just feel like for functions returning pointers or references, attaching the sigil to the function name feels wrong since the function name itself isn't a pointer/reference.

Reply:
I just noticed you did it without the space the other 3 times in this PR :) my OCD triggered on that :p